CacheMoney/frontend/src/config.js

```diff
@@ -0,0 +1,9 @@
+const config = {
+    url: "http://localhost:8080/",
+    // local: "http://localhost:8080/",
+    // remote: "http://cachemoney-env.eba-vtpiesgf.us-east-1.elasticbeanstalk.com/"
+    
+    url_img: "http://revature-cachemoney.s3-website-us-east-1.amazonaws.com/"
+    // local: "http://localhost:3000/"
+    // remote: "http://revature-cachemoney.s3-website-us-east-1.amazonaws.com/"
+};
```

Comment:
This file was added to the gitignore and deleted on main, you'll want to copy it to somewhere outside of the repo, delete it from the repo, commit the deletion and move the copy back.